test/babel-plugin-istanbul.js

```diff
@@ -148,5 +148,24 @@ describe('babel-plugin-istanbul', function () {
       })
       result.code.match(/statementMap/)
     })
+
+    it('should respect a changed cwd in options', function () {
```

Comment:
Could you take another look at the tests for this?

I tested your PR with the following steps:
1. Downloaded locally
2. Rebase onto current `master`.
3. Reverted your change to src/index.js.
4. Ran `npm test`.

The test passed without your change to `src/index.js`, leading me to believe that this test does not actually verify that your bug has been fixed.  Ideally I'd like to see a test that fails without your change to src/index.js.

Reply:
Hm. Yes, absolutely, this is meant to be a regression test. I’ll give it another shot. I do vaguely remember something occurring to me related to this, in the mean time, but I’ll need to take a deeper look to remember :)

Reply:
Ok, a bit of a rubber ducky discussion here... I've 'instrumented' the multi-project Jest setup that we are having issues with. I've hastily added the following code to the beginning of `makeShouldSkip`:

```js
function makeShouldSkip() {
  var exclude = void 0;
  var currentOpts
  return function shouldSkip(file, opts) {
    if(exclude && currentOpts && opts.cwd !== currentOpts.cwd) {
      console.log('Exclude exists but opts cwds are different, would need to reconstruct!')
      console.log('opts.cwd', opts.cwd)
      console.log('currentOpts.cwd', currentOpts.cwd)
    }
```

Which results in many cases of output like this:

```
Exclude exists but opts cwds are different, would need to reconstruct!
opts.cwd /Users/username/src/pubsweet-mrs/packages/logger
currentOpts.cwd /Users/username/src/pubsweet-mrs/packages/server
```

This would result in `/Users/username/src/pubsweet-mrs/packages/logger` being uncovered.

But in these tests, whenever I call `makeVisitor`, `exclude` is undefined, and gets rebuilt.

I'll figure it out, now that I've got the base issue in my head again.

Reply:
OK, test fixed in https://github.com/istanbuljs/babel-plugin-istanbul/pull/171/commits/5da84354ead52eccee312a90b81ac98107fb5984

It now accurately reflects what happens.